bci_tester/data.py

```diff
@@ -544,17 +551,14 @@ def create_BCI(
 
 REPOCLOSURE_CONTAINER = DerivedContainer(
     base="registry.fedoraproject.org/fedora:latest",
-    containerfile=r"""RUN dnf -y install 'dnf-command(repoclosure)'
+    containerfile=f"""RUN dnf -y install 'dnf-command(repoclosure)'
 RUN rm -f /etc/yum.repos.d/*repo
-RUN echo $'[SLE_BCI] \n\
-enabled=1 \n\
-name="SLE BCI" \n\
-autorefresh=0 \n\
-baseurl="""
-    + BCI_DEVEL_REPO
-    + r""" \n\
-priority=100' > /etc/yum.repos.d/SLE_BCI.repo
-""",
+RUN printf '[{BCI_REPO_NAME}]\\n\
+enabled=1\\n\
+name="SLE BCI"\\n\
+autorefresh=0\\n\
+baseurl={BCI_DEVEL_REPO}\\n\
+priority=100\\n' > /etc/yum.repos.d/{BCI_REPO_NAME}.repo""",
```

Comment:
```suggestion
RUN printf '[{BCI_REPO_NAME}] \n\
enabled=1 \n\
name="SLE BCI" \n\
autorefresh=0 \n\
baseurl={BCI_DEVEL_REPO} \n\
priority=100\n' > /etc/yum.repos.d/{BCI_REPO_NAME}.repo""",
```

Reply:
whats the point of that additional trailing whitespace? what am I missing?

Reply:
Just to reduce the diff noise. No point beyond that, so just ignore it.